Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -50,18 +53,47 @@ ModelProcessModelPlayerProxy::ModelProcessModelPlayerProxy(ModelProcessModelPlay
 
 ModelProcessModelPlayerProxy::~ModelProcessModelPlayerProxy()
 {
+    RELEASE_LOG(ModelElement, "%p - ModelProcessModelPlayerProxy deallocated id=%" PRIu64, this, m_id.toUInt64());
 }
 
 void ModelProcessModelPlayerProxy::invalidate()
 {
+    RELEASE_LOG(ModelElement, "%p - ModelProcessModelPlayerProxy invalidated id=%" PRIu64, this, m_id.toUInt64());
 }
 
-void ModelProcessModelPlayerProxy::loadModel()
+template<typename T>
+ALWAYS_INLINE void ModelProcessModelPlayerProxy::send(T&& message)
 {
-    // FIXME: Perform model loading here
-    m_webProcessConnection->send(Messages::ModelProcessModelPlayer::DidLoad(), m_id);
+    m_webProcessConnection->send(std::forward<T>(message), m_id);
+}
+
+void ModelProcessModelPlayerProxy::createLayer()
+{
+    m_layer = adoptNS([[CALayer alloc] init]);
+    [m_layer setBackgroundColor:cachedCGColor(Color::green).get()];
+    [m_layer setFrame:CGRectMake(0, 0, 100, 100)];
```

Comment:
Why this color and size? Is it for debug purposes?

Reply:
Mostly for debugging visualization until the whole feature is complete.
This PR deals only with CA hosting, so this is trying to put something in CA so that when we enable the model process code path, we will see model tags being rendered as green squares, proving that CA hosting from model process is working.